Automattic/jetpack
# VideoPress Block: Add adaptive progress bar color option

This PR adds a new toggle to the `Progress Bar Colors` settings of the video block that enables the new adaptive seekbar color setting that we've added to the VideoPress player:

<img width="1082" alt="Screen Shot 2021-11-29 at 11 25 46 AM" src="https://user-images.githubusercontent.com/789137/143922844-a846f84b-f9dd-47c7-915b-8e6c81c79658.png">

This setting is _disabled_ by default for now so that we can experiment with the feature. If we are happy with it we will likely make it enabled by default in a future PR (which will likely require a deprecation 😵‍💫).

#### Changes proposed in this Pull Request:
* Adds a new toggle to the VideoPress block settings that will enable the adaptive progress bar color.

#### Jetpack product discussion
pxWta-13K-p2

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Ensure you have a VideoPress plan on your site.
* Add a video to a post or edit an existing post that has a video block.
* In the sidebar, tick the `Match video` toggle. The 3 color settings for the progress colors should hide.
* Play the video, it should show the adaptive progress bar colors.
* Switch the toggle off again, and set some of the colors manually and click `Save colors`. The player should reload with the new colors.
